plugin/keyring_vault/tests/mtr/install_keyring_vault.test

```diff
@@ -0,0 +1,20 @@
+--disable_ps_protocol
```

Comment:
I am not sure. This test was originally added by a QA and I just did not notice this.